### PR TITLE
RALI - updates

### PR DESCRIPTION
**Pull request by kiritigowda**

### CODE REVIEW

**Comment by kiritigowda** [comment]
@LakshmiKumar23 & @paveltc no code change in this PR.

File: rali/source/device_manager.cpp [hunk 1]
```diff
@@ -44,19 +44,19 @@ CLProgram::~CLProgram()
     for(auto& kernel_pair : m_kernels)
         if(clReleaseKernel(kernel_pair.second)  != CL_SUCCESS)
             ERR("Could not release "+STR(kernel_pair.first))
-    m_kernels.clear();
+            m_kernels.clear();
```
Comment (LakshmiKumar23) [inline rali/source/device_manager.cpp:47]: @kiritigowda extra space

File: rali/source/device_manager.cpp [hunk 2]
```diff
@@ -140,21 +139,20 @@ DeviceManager::init_ocl(vx_context context)
...
+        cmd_queue = clCreateCommandQueueWithProperties(clcontext, dev_id, nullptr, &clerr);
 #else
-    cmd_queue = clCreateCommandQueue(opencl_context, dev_id, 0, &clerr);
+        cmd_queue = clCreateCommandQueue(opencl_context, dev_id, 0, &clerr);
 #endif
     if(clerr != CL_SUCCESS)
         THROW("clCreateCommandQueue failed " + TOSTR(clerr))
 
-    _resources.cmd_queue = cmd_queue;
+        _resources.cmd_queue = cmd_queue;
```
Comment (LakshmiKumar23) [inline rali/source/device_manager.cpp:155]: @kiritigowda extra space

File: rali/source/file_source_reader.cpp [hunk 3]
```diff
@@ -215,13 +213,13 @@ Reader::Status FileSourceReader::open_folder()
...
+        closedir(_src_dir);
     return Reader::Status::OK;
 }
 
 size_t FileSourceReader::get_file_shard_id()
 {
     if(_batch_count == 0 || _shard_count == 0)
         THROW("Shard (Batch) size cannot be set to 0")
-    return (_file_id / (_batch_count)) % _shard_count;
+        return (_file_id / (_batch_count)) % _shard_count;
```
Comment (LakshmiKumar23) [inline rali/source/file_source_reader.cpp:224]: @kiritigowda  extra space. Also on line 216

File: rali/source/image.cpp [hunk 4]
```diff
@@ -258,11 +258,11 @@ int Image::create_from_handle(vx_context context)
     vx_size size = (addr_in.dim_y+0) * (addr_in.stride_y+0);
 
     vx_df_image vx_color_format = interpret_color_fmt(_info._color_fmt);
-    vx_handle = vxCreateImageFromHandle(context, vx_color_format , &addr_in, ptr, vx_mem_type(_info._mem_type));
+    vx_handle = vxCreateImageFromHandle(context, vx_color_format, &addr_in, ptr, vx_mem_type(_info._mem_type));
     if((status = vxGetStatus((vx_reference)vx_handle)) != VX_SUCCESS)
         THROW("Error: vxCreateImageFromHandle(input:[" + TOSTR(_info.width()) + "x" + TOSTR(_info.height_batch()) + "]): failed " + TOSTR(status))
 
-    _info._type = ImageInfo::Type::HANDLE;
+        _info._type = ImageInfo::Type::HANDLE;
```
Comment (LakshmiKumar23) [inline rali/source/image.cpp:265]: @kiritigowda extra space

File: rali/source/image_loader.cpp [hunk 5]
```diff
@@ -142,13 +142,12 @@ ImageLoader::start_loading()
     if(!_is_initialized)
         THROW("start_loading() should be called after initialize() function is called")
 
-    _remaining_image_count = _image_loader->count();
+        _remaining_image_count = _image_loader->count();
```
Comment (LakshmiKumar23) [inline rali/source/image_loader.cpp:145]: @kiritigowda  extra space

File: rali/source/label_reader_folders.cpp [hunk 6]
```diff
@@ -165,6 +164,5 @@ void LabelReaderFolders::read_files(const std::string& _path)
     }
     if(_file_names.empty())
         WRN("LabelReader: Could not find any file in " + _path)
-    closedir(_src_dir);
+        closedir(_src_dir);
```
Comment (LakshmiKumar23) [inline rali/source/label_reader_folders.cpp:167]: @kiritigowda extra space

File: rali/source/node_resize.cpp [hunk 7]
```diff
@@ -46,18 +45,18 @@ void ResizeNode::create_node()
 
     width_status = vxAddArrayItems(_dst_roi_width, _batch_size, dst_roi_width.data(), sizeof(vx_uint32));
     height_status = vxAddArrayItems(_dst_roi_height, _batch_size, dst_roi_height.data(), sizeof(vx_uint32));
-     if(width_status != 0 || height_status != 0)
+    if(width_status != 0 || height_status != 0)
         THROW(" vxAddArrayItems failed in the resize (vxExtrppNode_ResizebatchPD) node: "+ TOSTR(width_status) + "  "+ TOSTR(height_status))
 
-   _node = vxExtrppNode_ResizebatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _dst_roi_width, _dst_roi_height, _batch_size);
+        _node = vxExtrppNode_ResizebatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _dst_roi_width, _dst_roi_height, _batch_size);
```
Comment (LakshmiKumar23) [inline rali/source/node_resize.cpp:51]: @kiritigowda extra space

File: rali/source/node_resize_crop_mirror.cpp [hunk 8]
```diff
@@ -51,14 +51,14 @@ void ResizeCropMirrorNode::create_node()
     height_status = vxAddArrayItems(_dst_roi_height, _batch_size, dst_roi_height.data(), sizeof(vx_uint32));
     if(width_status != 0 || height_status != 0)
         THROW(" vxAddArrayItems failed in the crop resize node (vxExtrppNode_ResizeCropbatchPD    )  node: "+ TOSTR(width_status) + "  "+ TOSTR(height_status))
-    _mirror.create_array(_graph ,VX_TYPE_UINT32, _batch_size);
+        _mirror.create_array(_graph,VX_TYPE_UINT32, _batch_size);
```
Comment (LakshmiKumar23) [inline rali/source/node_resize_crop_mirror.cpp:54]: @kiritigowda extra space

File: rali/source/node_rotate.cpp [hunk 9]
```diff
@@ -48,14 +48,14 @@ void RotateNode::create_node()
     if(width_status != 0 || height_status != 0)
         THROW(" vxAddArrayItems failed in the resize (vxExtrppNode_ResizebatchPD) node: "+ TOSTR(width_status) + "  "+ TOSTR(height_status))
 
-    _angle.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
-   _node = vxExtrppNode_RotatebatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _dst_roi_width, _dst_roi_height, _angle.default_array(), _batch_size);
+        _angle.create_array(_graph, VX_TYPE_FLOAT32, _batch_size);
+    _node = vxExtrppNode_RotatebatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _dst_roi_width, _dst_roi_height, _angle.default_array(), _batch_size);
```
Comment (LakshmiKumar23) [inline rali/source/node_rotate.cpp:52]: @kiritigowda extra spaces

File: rali/source/node_warp_affine.cpp [hunk 10]
```diff
@@ -65,15 +65,15 @@ void WarpAffineNode::create_node()
     if(width_status != 0 || height_status != 0)
         THROW(" vxAddArrayItems failed in the rotate (vxExtrppNode_WarpAffinePD) node: "+ TOSTR(width_status) + "  "+ TOSTR(height_status))
 
-    vx_status status;
+        vx_status status;
```
Comment (LakshmiKumar23) [inline rali/source/node_warp_affine.cpp:68]: @kiritigowda extra space

**Comment by LakshmiKumar23** [comment]
@kiritigowda There were a bunch of extra lines/spaces. I have marked them

**Comment by kiritigowda** [comment]
@LakshmiKumar23 & @paveltc I am going to close this PR as it might compromise some conditional statements without braces. I will reopen another PR.

**Pull request closed without merging**